crates/sncast/src/starknet_commands/account/create.rs

```diff
@@ -314,3 +320,16 @@ fn write_account_to_file(
     )?;
     Ok(())
 }
+
+fn generate_deploy_command(accounts_file: &Utf8PathBuf, rpc_url: &str, account: &str) -> String {
+    let accounts_flag = if accounts_file.to_string().contains(DEFAULT_ACCOUNTS_FILE) {
+        format!(" --accounts-file {accounts_file}")
```

Comment:
Shouldn't this one be only displayed if non default accounts file is used?

Reply:
+ I'd handle it in a simpler manner: if user passed `--accounts-file` then show the command that uses it, if not, then do not show anything.

Reply:
It looks like this:
without account file 
command:
```shell
sncast account create --name test1 -c 0x61dac032f228abef9c6626f995015233097ae253a7f72d68552db02f2971b8f --type oz --add-profile test1
```

output:
```shell
command: account create
add_profile: Profile test1 successfully added to snfoundry.toml
address: 0x0440ffd0fffbcdf12bc0ac683a0d1d7221336f5e67894346903deb23511722bb
max_fee: 30100000000000
message: Account successfully created. Prefund generated address with at least <max_fee> STRK tokens or an equivalent amount of ETH tokens. It is good to send more in the case of higher demand.

After prefunding the address, run:
sncast account deploy --url http://127.0.0.1:5050 --name test1 --fee-token strk

To see account creation details, visit:
account: https://sepolia.starkscan.co/contract/0x440ffd0fffbcdf12bc0ac683a0d1d7221336f5e67894346903deb23511722bb
```

with account file
command:
```shell
sncast --accounts-file konta.json account create --name test2 -c 0x61dac032f228abef9c6626f995015233097ae253a7f72d68552db02f2971b8f --type oz --add-profile test2
```

output:
```shell
command: account create
add_profile: Profile test2 successfully added to snfoundry.toml
address: 0x01d1efd96c297f2383ce5debf7353ad54852c2673d1b033f1a94f2e50a4f4b60
max_fee: 30100000000000
message: Account successfully created. Prefund generated address with at least <max_fee> STRK tokens or an equivalent amount of ETH tokens. It is good to send more in the case of higher demand.

After prefunding the address, run:
sncast --accounts-file konta.json account deploy --url http://127.0.0.1:5050 --name test2 --fee-token strk

To see account creation details, visit:
account: https://sepolia.starkscan.co/contract/0x1d1efd96c297f2383ce5debf7353ad54852c2673d1b033f1a94f2e50a4f4b60
```